core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -300,25 +297,25 @@ private boolean putInternal(PageId pageId, byte[] page) {
       }
     }
 
-    Pair<ReadWriteLock, ReadWriteLock> pageLockPair = getPageLockPair(pageId, victim);
+    Pair<ReadWriteLock, ReadWriteLock> pageLockPair =
+        getPageLockPair(pageId, victimPageInfo.getPageId());
     try (LockResource r1 = new LockResource(pageLockPair.getFirst().writeLock());
-        LockResource r2 = new LockResource(pageLockPair.getSecond().writeLock())) {
+         LockResource r2 = new LockResource(pageLockPair.getSecond().writeLock())) {
       try (LockResource r3 = new LockResource(mMetaLock.writeLock())) {
         if (mMetaStore.hasPage(pageId)) {
           LOG.debug("{} is already inserted by a racing thread", pageId);
           // TODO(binfan): we should return more informative result in the future
           return true;
         }
-        if (!mMetaStore.hasPage(victim)) {
+        if (!mMetaStore.hasPage(victimPageInfo.getPageId())) {
```

Comment:
Should we change the metric below to be more specific?

Reply:
count it as `PUT_RACING_EVICTION_ERRORS`